src/main/java/org/myrobotlab/service/meta/Py4jMeta.java

```diff
@@ -22,8 +22,8 @@ public Py4jMeta() {
 
     // Used just as a Python exe redistributable.
     // ABSOLUTELY NO JNI/JNA IS USED
-    addDependency("org.bytedeco", "cpython-platform", "3.10.8-1.5.8");
-    addDependency("org.bytedeco", "cpython", "3.10.8-1.5.8");
+    addDependency("org.bytedeco", "cpython-platform", "3.11.3-1.5.9");
```

Comment:
As far as I am aware, upgrading this will break Py4j on raspberry pis because the arm hf and aarch64 artifacts are not listed as dependencies of cpython-platform for 3.11, we would need to add an explicit dependency on the arm 64 and armhf classifiers of the package, which we don't support in our ivy and maven wrappers